solr/core/src/java/org/apache/solr/handler/loader/JsonLoader.java

```diff
@@ -593,14 +602,14 @@ private void parseExtendedFieldValue(int ev, SolrInputField sif) throws IOExcept
       Object normalFieldValue = null;
       Map<String, Object> extendedInfo = null;
 
-      for (SolrInputField field: extendedSolrDocument) {
-        Object fieldVal = field.getValue();
-        String fieldName = field.getName();
-        if ("boost".equals(fieldName)) {
-          Object boostVal = fieldVal;
+      for (SolrInputField entry: extendedSolrDocument) {
+        Object label = entry.getValue();
```

Comment:
LOL you have these mixed up :-)  entry.getName should be the label.

Reply:
oops